cli/src/main/resources/nls/Help.properties

```
@@ -62,6 +62,8 @@ cmd.kotlinc=Tool commandlet for Kotlin (compiler for JRE language).
 cmd.kotlinc.detail=Kotlin Compiler (kotlinc) is the command-line tool for compiling Kotlin code. Detailed documentation can be found at https://kotlinlang.org/docs/home.html
 cmd.kotlincnative=Tool commandlet for Kotlin-Native (compiler for JRE language).
 cmd.kotlincnative.detail=Kotlin/Native Compiler (kotlincnative) compiles Kotlin code to native executables. Detailed documentation can be found at https://kotlinlang.org/docs/reference/native-overview.html
+cmd.kubectl=Tool commandlet for kubernetes. Detailed documentation can be found at https://kubernetes.io/docs/home/
+cmd.kubectl.detail=The kubectl commandlet allows to install and use kubernetes. On Windows WSL 2 (Windows Subsystem for Linux) has to be installed properly as a prerequisite. The setup on windows will then install kubernetes with K3D. K3D will create a cluster with a single node with a default name as "ideasy-cluster"
```

```suggestion
cmd.kubectl.detail=The kubectl commandlet allows to install and use kubernetes. On Windows WSL 2 (Windows Subsystem for Linux) has to be installed properly as a prerequisite.
```

I suggest to remove the last 2 sentences since it is wrong. Rancher-Desktop or Docker-Desktop installs kubernetes. What "implementation" they use will depended on the edition (rancher vs. docker desktop) and already changed over time so it also depends on the version. Also I am not aware that we create a custom cluster name anywhere.